app/controllers/api/v1/venues_controller.rb

```diff
@@ -0,0 +1,59 @@
+# :reek:InstanceVariableAssumption :reek:NilCheck
+class Api::V1::VenuesController < ApplicationController
+  before_action :organizer?, only: %i[create update destroy]
+  before_action :set_venue, only: %i[show update destroy]
+
+  def index
+    @venues = Venue.all.reverse
+    render json: @venues
+  end
+
+  def show
+    if @venue
```

Comment:
what do you check here?
i have already done this in method set_venue
```
return head :not_found if @venue.nil?
```

Reply:
done